README.md

```diff
@@ -21,13 +21,14 @@ Install the latest verison of .NET Core for Windows/Linux or Mac.
 
 1. Install SQL Server Developer 2019
     > https://www.microsoft.com/en-us/sql-server/sql-server-downloads
-2. Update connection string server name and credentials in [appsettings.json](./src/BlackSlope.Api/appsettings.json)
+2. Run Create DATABASE movies command to create the database locally .Update connection string server name and 
```

Comment:
You do not need to create the database first, ef migration process should create it.

Reply:
I'm not too familiar with EF Migration, but I had to manually create my Movies database for the db-update script to succeed.

Reply:
It should not be the case. What is your DB server?

Reply:
A localdb on SQL Server 2019

Reply:
This work has been patched in prior PR's.

[See here.](https://github.com/SlalomBuild/blackslope.net/pull/46/files)